srcpkgs/sonic-visualiser/template

```diff
@@ -1,55 +1,25 @@
 # Template file for 'sonic-visualiser'
 pkgname=sonic-visualiser
-version=4.2
-revision=2
-build_style=gnu-configure
-build_helper=qmake
-hostmakedepends="pkg-config capnproto-devel"
+version=4.4
+revision=1
+build_style=meson
+hostmakedepends="pkg-config capnproto-devel qt5-host-tools"
 makedepends="capnproto-devel jack-devel libfishsound-devel libid3tag-devel
  liblo-devel liblrdf-devel libmad-devel liboggz-devel libsamplerate-devel
  opusfile-devel portaudio-devel pulseaudio-devel qt5-svg-devel rubberband-devel
- sord-devel vamp-plugin-sdk-devel"
+ speex-devel sord-devel vamp-plugin-sdk-devel"
 short_desc="Viewing and analysing the contents of music audio files"
 maintainer="newbluemoon <blaumolch@mailbox.org>"
 license="GPL-2.0-or-later"
 homepage="https://www.sonicvisualiser.org/"
-changelog="https://code.soundsoftware.ac.uk/projects/sonic-visualiser/repository/entry/CHANGELOG"
-distfiles="https://code.soundsoftware.ac.uk/attachments/download/2755/${pkgname}-${version}.tar.gz"
-checksum=38e91a4035962ad177905a9f1249136fe88d4b596fd9b8f40c82b60dadca5bf3
+changelog="https://github.com/sonic-visualiser/sonic-visualiser/blob/default/CHANGELOG"
+distfiles="https://github.com/sonic-visualiser/sonic-visualiser/releases/download/sv_v${version}/sonic-visualiser-${version}.tar.gz"
+checksum=762d95cfceff5f2ba31bb632596f6c822ca5b4ee21f3e3615add20bfc24ae16e
 
 if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 	makedepends+=" libatomic-devel"
-fi
 
-if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" qt5-qmake qt5-svg-devel"
-	configure_args+=" ac_cv_prog_QMAKE=${XBPS_WRAPPERDIR}/qmake"
+	post_extract() {
+		vsed -i "s/^general_link_args = \[\]/general_link_args = \['-latomic'\]/" meson.build
```

Comment:
Use `"s@...@...@"` so you don't need so much escaping.

Reply:
Ah, and don't define the function inside the conditional block... Just define the function and check `XBPS_TARGET_NOATOMIC8` inside it.

Reply:
Regarding the conditional block: just wanted to save a bit space, but looking at it again it did seem a bit strange... ;)

But using `@` as a delimiter in the sed command doesn’t work, because the curly brackets need to be escaped, not the slash. :)

Reply:
Don't review PRs while sleepy :P:P